### ISSUE TITLE
Open fellowship applications - 4/10/17

### ISSUE THREAD

**Issue by auremoser** [mozilla/science.mozilla.org#607]
The Fellows application is open as of Monday, April 10th!

Can we:

- [ ] Add the 'apply to be a fellow' hero images on the homepage (same as last year), linking to [this application](https://mozilla-science-lab.forms.fm/mozilla-fellows-for-science-2017)
- [ ] Add the 'Apply Now' button on https://science.mozilla.org/programs/fellowships, linking to [this application](https://mozilla-science-lab.forms.fm/mozilla-fellows-for-science-2017)
- [ ] Add the 'Apply Now' block at the bottom of https://science.mozilla.org/programs/fellowships, linking to [this application](https://mozilla-science-lab.forms.fm/mozilla-fellows-for-science-2017)

I've staged some changes as PR (#606) to update the 
* FAQ text 
* Landing page content for 2017 CFP

If this could go live on Monday, April 10th, 9am ET that would be lovely, but please let me know if there are problems with this timeline. Thank you!

**Comment by akshatnitd** [comment]
@auremoser @alanmoo @acabunoc For the first checkpoint, do we need to design it like in [this](https://redpen.io/jv7dd38e552814c20a) mockup ?

**Comment by dhiraj161298** [comment]
@akshatnitd The first checkpoint could be completed by following the [commit](https://github.com/mozilla/science.mozilla.org/commit/367c461a355234804df5015752ddb5dba68776f6) which removed the 'Apply Now' section.

**Comment by akshatnitd** [comment]
@dhiraj161298 Okay cool! Making the required changes.

**Comment by acabunoc** [comment]
Thanks @akshatnitd

**Comment by acabunoc** [comment]
Quick note for @akshatnitd -- please base your changes off the work @auremoser has already done in https://github.com/mozilla/science.mozilla.org/pull/606. This way we can easily merge in for launch!

Thanks again 😸

**Comment by akshatnitd** [comment]
@acabunoc Sure, will take care of that!

**Comment by akshatnitd** [comment]
@acabunoc @auremoser What text needs to be written on the 'Apply Now' block at the bottom apart from the button?

**Comment by acabunoc** [comment]
@akshatnitd I think it should be similar to last year.

`Deadline for applications is May 14, 2017`

**Comment by akshatnitd** [comment]
@acabunoc Okay :+1:

**Comment by hannahkane** [comment]
@cadecairos @gideonthomas @alanmoo - are any of you East Coasters available to push these changes live before 9am on Monday? Or, can we schedule this in advance as @auremoser suggested?

**Comment by akshatnitd** [comment]
@acabunoc I have made the necessary changes. Should I send a PR or does #606 needs to be merged first?

**Comment by acabunoc** [comment]
Hey Akshat -- feel free to send along a PR that also includes #606
